src/assets/TestFramework/Deployment.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
What does this used for?
The files in this directory might also be shipped as part of the generated SDK - please double confirm.
And some of the things in this directory is synced (or checked if synced) by the pipeline

Reply:
These files are faked to make testProjects using "armtemplate deployment" compilable. 
https://github.com/changlong-liu/autorest.csharp/blob/20220222-signalr/test/TestProjects/MgmtSignalR/tests/Generated/Scenario/SignalRTest.cs#L50

Is there a possibility to exclude some assets out of generated SDK?